### ISSUE TITLE
X1-AV-39 Pass Initial values into airbrakes

### ISSUE THREAD

**Issue by klemie** [UVicRocketry/Xenia1-flight-Computer#75]
**Description**
Initial values are needed inorder to create a prediction. 

**Acceptance Criteria**
 <Write specifically what is need for this issue to be completed>

**Dev Notes**
[for assignee]<write down your thought process when developing this issue>

**Linked issues**
 <If there are any related issues>

<Add to milestone>

**Comment by jjderooy** [comment]
@PokeyOne is this required for suborbit as it is written currently?

**Comment by PokeyOne** [comment]
@klemie What are initial values?

**Comment by PokeyOne** [comment]
I think I kinda understand this. There will be some values that the script will need, such as launch altitude. During flight it will also need current altitude, velocity, and acceleration if available. If some of those values are not available directly, using some sort of estimate may be viable. There may be some things that could be done with a temperature and pressure reading on the ground, but that is not currently how it is written.

In summary:

Initial values:
- [ ] launch altitude

During flight
- [ ] altitude
- [ ] velocity
- [ ] acceleration

**Comment by klemie** [comment]
I need to fill out this issue better
